randovania/games/samus_returns/logic_database/Area 2 - Dam Exterior.txt

```diff
@@ -794,11 +807,7 @@ Extra - asset_id: collision_camera_032
   > Door to Metroid Caverns Lobby
       Varia Suit
   > Pickup (Missile Tank)
-      All of the following:
-          Varia Suit
-          Any of the following:
-              Knowledge (Beginner) and Disabled Buff Beam Burst and Lay Power Bomb
-              Lay Bomb and Shoot Beam Burst
+      Varia Suit and Destroy Blob Throwers/Steel Orbs and Lay Any Bomb
```

Comment:
Weird unnecessary grouping in the DB (pretty print is fine)
![image](https://github.com/randovania/randovania/assets/38679103/e0b6e26b-5827-44c4-a550-71193470abc3)